boards/arm/nucleo_wb55rg/nucleo_wb55rg_defconfig

```diff
@@ -3,6 +3,14 @@ CONFIG_SOC_STM32WB55XX=y
 # 32MHz system clock
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32000000
 
+CONFIG_STM32_LPTIM_TIMER=y
+CONFIG_STM32_LPTIM_CLOCK_LSE=y
+CONFIG_STM32_LPTIM_CLOCK_LSI=n
+# CONFIG_DEBUG=y
+CONFIG_TICKLESS_KERNEL=y
+CONFIG_TIMESLICING=n
```

Comment:
Can you elaborate why you're changing these 2 settings (as I guess this is not anecdotal)